src/runtime_src/core/pcie/driver/linux/xocl/lib/libqdma/qdma_request.c

```diff
@@ -1,7 +1,7 @@
 /*
  * This file is part of the Xilinx DMA IP Core driver for Linux
  *
- * Copyright (c) 2017-present,  Xilinx, Inc.
+ * Copyright (c) 2017-present,  Advanced Micro Devices, Inc.
```

Comment:
Please follow same approach in all the files as mentioned above, * - 2022 Xilinx and 2022 - 2024 Advanced Micro Devices